requirements.txt

```diff
@@ -7,4 +7,6 @@ Sphinx-Substitution-Extensions == 2020.9.30.0
 sphinx-sitemap == 2.2.0
 sphinx-togglebutton === 0.3.2
 sphinxcontrib-images === 0.9.4
-myst-parser === 0.18.0
\ No newline at end of file
+myst-parser === 0.18.0
+linkify === 1.4
+linkify-it-py === 2.0.0
```

Comment:
I'll just note - I do not know why `linkify` did not work. `linkify-it-py` *did* seem to work. I'll remove `linkify` in another pass with a bit more testing. But wanted to confirm this works on other machines first.